CHANGELOG.md

```diff
@@ -30,6 +32,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Spot shadows cause light range cutoff (#1312, **@tomas7770**).
 
 
+### Changed
+
+- Move logger to telemetry namespace (#1265, **@roby2014**).
+- Move metrics to telemetry namespace (#1265, **@roby2014**).
```

Comment:
This should be moved to the previous ### Changes section, before the ### Fixed section.
Also, I think you can merge those two, specially considering they come from the same issue.